src/pyramid/scaffolds/tests.py

```diff
@@ -28,7 +28,7 @@ def install(self, tmpl_name):  # pragma: no cover
             self.directory = tempfile.mkdtemp()
             self.make_venv(self.directory)
             here = os.path.abspath(os.path.dirname(__file__))
-            os.chdir(os.path.dirname(os.path.dirname(here)))
+            os.chdir(os.path.dirname(os.path.dirname(os.path.dirname(here))))
```

Comment:
This looks weird to me. Why is it called three times? Should this be `os.chdir(os.path.dirname(here))`?

Reply:
... sorry that it looks weird but I have faith that if you look at it again you’ll see the errors in this comment.